templates/new_service/.github/workflows/build-nocache.yml

```diff
@@ -0,0 +1,45 @@
+name: Build No Cache
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: '30 12 * * 0'
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  build-no-cache:
+    env:
+      DOCKER_IMAGE: #DOCKER_REPOSITORY#
+    outputs:
+      docker-image-tag: ${{ steps.build-image.outputs.tag }}
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      name: Checkout
+
+    - name: Build without cache and push docker image
+      id: build-image
+      uses: DFE-Digital/github-actions/build-docker-image@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        context: .
+        docker-repository: ${{ env.DOCKER_IMAGE }}
+        max-cache: true
+        reuse-cache: false
+        snyk-token: ${{ secrets.SNYK_TOKEN }}
```

Comment:
We should document the requirement for this secret
Can we detail how to use the templates in https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-service.md#deploy-via-github-actions ?

Reply:
Done